ddtrace/settings/integration.py

```diff
@@ -61,6 +52,48 @@ def __init__(self, global_config, name, *args, **kwargs):
         # unified.
         self.setdefault("service_name", service)
 
+    def _get_analytics_settings(self):
+        # We can have deprecated names for integrations used for analytics settings
+        deprecated_name = self._deprecated_name if hasattr(self, "_deprecated_name") else None
```

Comment:
Just my opinion here but I think it'd be a lot easier to read through this logic if the conditionals weren't inlined. It might be a lot more verbose but it'd be worth it IMO

eg. 
```python
var = None
if ......:
   var = os.environ.get(...)
```

instead of 

```python
var = os.environ.get(...) if ..... else None or os.environ.get(...) if ... else None
```